src/languages/es.ts

```diff
@@ -1879,6 +1879,7 @@ export default {
             addRate: 'Añadir tasa',
             workspaceDefault: 'Moneda por defecto del espacio de trabajo',
             foreignDefault: 'Moneda extranjera por defecto',
+            customTaxName: 'Nombre del impuesto personalizado',
```

Comment:
this needs to be updated based on latest in Slack

Reply:
Done